src/controller/python/BUILD.gn

```diff
@@ -222,40 +305,69 @@ pw_python_action("python") {
   }
 
   if (current_os == "mac") {
-    platform_tag = "macosx_" + string_replace(mac_deployment_target, ".", "_")
+    py_platform_tag = "macosx_" + string_replace(mac_deployment_target, ".", "_")
   } else {
-    platform_tag = current_os
+    py_platform_tag = current_os
   }
 
-  platform_tag = platform_tag + "_" + cpu_tag
+  py_platform_tag = py_platform_tag + "_" + cpu_tag
 
-  tags = "cp37-abi3-" + platform_tag
-
-  args = [
-    "--package_name",
-    "chip",
-    "--build_number",
-    "0.0",
-    "--build_dir",
-    rebase_path("${target_gen_dir}/${target_name}.py_build", root_build_dir),
-    "--dist_dir",
-    rebase_path(_dist_dir, root_build_dir),
-    "--manifest",
-    rebase_path(_py_manifest_file, root_build_dir),
-    "--plat-name",
-    platform_tag,
-  ]
+  tags = "cp37-abi3-" + py_platform_tag
 
   if (chip_controller) {
+    lib_name = "_ChipDeviceCtrl.so"
   } else {
-    args += [
-      "--server",
-      "True",
-    ]
+    lib_name = "_ChipServer.so"
   }
 
+  py_package_name = "chip-library"
+
   public_deps = [ ":ChipDeviceCtrl" ]
 
-  output_name = "chip-0.0-${tags}.whl"
-  outputs = [ "${_dist_dir}/$output_name" ]
+  output_name = "chip_library-${chip_python_version}-${tags}.whl"
+}
+
+chip_python_wheel_action("chip-repl") {
```

Comment:
Shouldn't this have a dependency on `chip-library`?

Reply:
Added a runtime dependency using `data_deps`.